addons/binding/org.openhab.binding.homematic/META-INF/MANIFEST.MF

```diff
@@ -40,8 +40,14 @@ Import-Package: com.google.common.collect,
  org.eclipse.smarthome.core.thing.type,
  org.eclipse.smarthome.core.types,
  org.jupnp.model.meta,
+ org.openhab.binding.homematic,
+ org.openhab.binding.homematic.handler,
+ org.openhab.binding.homematic.type,
  org.osgi.framework,
  org.osgi.service.component,
  org.osgi.service.component.annotations;resolution:=optional,
  org.slf4j
+Export-Package: org.openhab.binding.homematic,
+ org.openhab.binding.homematic.handler,
+ org.openhab.binding.homematic.type
```

Comment:
Should we export this one?

Reply:
I'm not sure, but this was required for the 1.x Homematic Action. But the Action is not in use anymore, so you can remove the export.